scripts/devaccess/devaccess_users.yaml

```diff
@@ -1,14 +1,179 @@
+---
 roles:
 - name: ALL
   commands:
-  - oc get nodes -ndefault
-  - oc get nodes -ndefault -ojson
-  - rpm -qa
-  - oc get pods -ndefault
-  - oc get pods -nopenshift-infra
-  - oc get pods -nlogging
-  - oc get routes -ndefault
-  - oc get routes -nlogging
+    - base: oc get
+      types:
+        - names:
+            - node
+            - nodes
+          runner: oc_get_generic
+          output:
+            - json
+        - names:
+            - pod
+            - pods
+          runner: oc_get_generic
+          namespaces:
+            - default
+            - logging
+            - openshift-infra
+        - names:
+            - route
+            - routes
+          runner: oc_get_routes
+          namespaces:
+            - default
+            - logging
+        - names:
+            - storageclass
+          runner: oc_get_generic
+        - names:
+            - pvc
+          runner: oc_get_generic
+          namespaces:
+            - all
+        - names:
+            - ns
+            - project
+            - projects
+            - namespace
+            - namespaces
+          runner: oc_get_generic
+        - names:
+            - svc
+            - service
+            - services
+          runner: oc_get_generic
+          namespaces:
+            - all
+
+    - base: oc describe
+      types:
+        - names:
+            - clusterresourcequota
+
+    - base: rpm
+      runner: safe_command
+      switches:
+        - -qa
+
+    - base: cat
+      runner: super_safe_command
+      leftover:
+        - /proc/cpuinfo
+        - /proc/meminfo
+        - /proc/vmstat
+        - /proc/interrupts
+        - /proc/slabinfo
+        - /proc/cmdline
+        - /etc/origin/node/node-config.yaml
+
+    - base: ethtool
+      runner: safe_command
+      switches:
+        - -i
+      leftover:
+        - '^eth[0-9]$'
+        - '^enp\ds\d$'
+    - base: uptime
+      runner: safe_command
+    - base: lsblk
+      runner: safe_command
+      switches:
+        - -at
+        - --output
+      leftover:
+        - NAME,KNAME,FSTYPE,RA,SIZE,SCHED,TYPE
+    - base: lscpu
+      runner: safe_command
+    - base: uname
+      runner: safe_command
+      switches:
+        - -r
+    - base: df
+      runner: safe_command
+      switches:
+        - -h
+    - base: iostat
+      runner: cmd_with_tail
+      switches:
+        - -x
+        - -y
+    - base: mpstat
+      runner: cmd_with_tail
+      switches:
+        - -P
+        - ALL
+      leftover:
+        - '^\d \d+$'
+    - base: vmstat
+      runner: cmd_with_tail
+      switches:
+        - -w
+      leftover:
+        - '^\d \d+$'
```

Comment:
Suggest delay (first value) should accept more than just \d.  A 10 or 30 second delay is appropriate.